Contributors-Guide/Bug-report-Life-Cycle.md

```diff
@@ -36,6 +36,8 @@ This page describes the life of a bug report.
     -   **CLOSED/CURRENTRELEASE** when a code change that fixes the
```

Comment:
We only use CLOSED/CURRENTRELEASE when the patch is merged in the release-branch for the version that this bug is reported against, **and** a version (tagged) is released with the fix.

Could you update this as well?